### ISSUE TITLE
Running on multiple gpus

### ISSUE THREAD

**Issue by Xujingkk** [TIO-IKIM/CellViT#11]
I'm very interested in your research. Constrained by gpu resources, I want to run code in parallel on multiple gpu cards. I added "model = nn.DataParallel(model, device_ids=[0, 1])" but it didn't work and I got the following error. I have made multiple attempts but haven't solved it yet and would appreciate your help! Also the paper mentions three-fold cross validation, is it using three folds of the original pannuke dataset for the training set, validation set and test set respectively? In the original pannuke dataset, Fold 1 contains 2,657 images, Fold 2 contains 2,524 images and Fold 3 contains 2,723 images. In that case, if the second fold is used for the training set, then the training set will be less than the validation set and the test set, will this have no effect on the results? Maybe my interpretation is wrong, please correct me.
![0da268766aece4450cf4b5c3a80fef8](https://github.com/TIO-IKIM/CellViT/assets/131575723/0c3cfa3e-bd77-4589-b7b5-8fb8b0448090)

**Comment by FabianHoerst** [comment]
Have you tried running it on one GPU with smaller batch size to check of the error occurs when running without data parallel?

Sure the folds should have an effect, but the results are averaged in the paper. For the final models, we performed training on all 3 folds.
